experimental/packages/otlp-transformer/test/metrics.test.ts

```diff
@@ -70,7 +70,8 @@ describe('Metrics', () => {
           valueType: ValueType.INT,
         },
         aggregationTemporality,
-        dataPointType: DataPointType.SINGULAR,
+        dataPointType: DataPointType.SUM,
+        isMonotonic: true,
```

Comment:
Might be worth adding a non-monotonic case for this.

Reply:
Good point! Added in [2b40e9f](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/2b40e9f381f380e71d796b4401921611cd38239a) :slightly_smiling_face: